addon/components/variable-number/number.ts

```diff
@@ -44,7 +44,18 @@ export default class VariableNumberPluginNumberComponent extends Component<Args>
     return this.args.node;
   }
   get formattedNumber() {
-    return this.node.attrs.value as string;
+    const writtenNumber = Boolean(this.writtenNumber);
```

Comment:
why is this needed? Maybe because the default value isn't a boolean? (see other comment)